### PR TITLE
[MI-2669_1]: Added a top level CSS class as a unique selector

### PR DESCRIPTION
**Pull request by avas27JTG**
Added a top level CSS class as a unique selector

### CODE REVIEW

File: webapp/src/styles/_utils.scss [hunk 1]
```diff
@@ -1,177 +1,179 @@
...
-    margin-bottom: 25px;
-}
+    .margin-bottom-25 {
+        margin-bottom: 25px;
+    }
 
-.margin-top-80 {
-    margin-top: 80px
-}
+    .margin-top-80 {
```
Comment (sooraj-shukla) [inline webapp/src/styles/_utils.scss:19]: Aren't we using `mm-ui-library`?

File: webapp/src/styles/_global.scss [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+        overflow: hidden;
+    }
+    
+    .button-wrapper {
+        flex-basis: 30%;
     }
+    
+    .rhs-wrapper {
+        display: flex;
+        flex-direction: column;
```
Comment (sooraj-shukla) [inline webapp/src/styles/_global.scss:19]: We can use util classes for this.

File: webapp/src/styles/_components.scss [hunk 3]
```diff
@@ -1,76 +1,78 @@
...
+        }
+    }
+    
+    .rhs-filter-popover {
+        width: calc(100% - 30px);
+        padding: 20px;
         position: absolute;
-        border: 2px solid var(--center-channel-bg);
-        background-color: var(--link-color);
+        height: fit-content;
```
Comment (sooraj-shukla) [inline webapp/src/styles/_components.scss:53]: `height:fit-content`  isn't supported on firefox.
image

File: webapp/src/styles/_components.scss [hunk 4]
```diff
@@ -1,76 +1,78 @@
...
+        }
+    }
+    
+    .rhs-project-list-wrapper {
+        margin-top: auto;
+        padding: 12px;
+        width: 100%;
+        background-color: var(--center-channel-bg);
+        box-shadow: 5px -1px 5px rgba(0, 0, 0, 0.1);
+        text-align: center;
```
Comment (sooraj-shukla) [inline webapp/src/styles/_components.scss:29]: We can use util classes.

File: webapp/src/styles/_components.scss [hunk 5]
```diff
@@ -1,76 +1,78 @@
...
-    &--error {
-        font-size: 58px;
-        color: var(--error-text);
+    
+    .project-details-unlink-button {
+        max-width: 42px;
+        height: 34px;
+        margin-right: 16px;
+        display: block;
+        margin-left: auto;
```
Comment (sooraj-shukla) [inline webapp/src/styles/_components.scss:11]: Shorthand for margin.

**Review by sooraj-shukla — CHANGES_REQUESTED** [review]

File: webapp/src/styles/_global.scss (hunk 2 above)
Comment (avas27JTG) [reply webapp/src/styles/_global.scss:19]: this is a global style and here we have just overridden some stylings of different classes meaning if any class of this file is used anywhere within the code then we are targeting them and adding some common/global styles at just one place

putting utils classes in such cases will tend to add that util class at all those places wherever needed

File: webapp/src/styles/_utils.scss (hunk 1 above)
Comment (avas27JTG) [reply webapp/src/styles/_utils.scss:19]: not for azure devops

**Review by sooraj-shukla — APPROVED** [review]

**Pull request merged**